Docs/source/Hardware/Boards.rst

```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
     :align: center
 
     LTC6811 Block Diagram 
+
+
+
+BPS Minion Board
+================
+
+Overview
+^^^^^^^^
+`GitHub Link <https://github.com/lhr-solar/BPS-MinionPCB>`__
+
+
+
+Brief Description/Purpose:
+    The BPS Minion Board measures the voltage and temperature of the batteries and sends the data to the BPS Leader Board. 
+    There are four temperature connector minion boards. Each board can measure 12 modules and 16 temperature sensors. 
+
+Pertinent Regulations
+^^^^^^^^^^^^^^^^^^^^^
+===========  ============================================== ===========================================================
+Regulation   Description of Regulation                      How Regulation is Met
+
+**8.3.A.5**  | System in which measurements are constantly  | The Minion board constantly takes voltage and temperature 
+             | and where actions are taken immediately      | measurements of the battery modules and transmits data
+             | without operator intervention to open the    | back to the leader board.
+             | Main Power Switch should a battery  
+             | Protection Fault occur. 
+===========  ============================================== ===========================================================
+
+Context
+^^^^^^^
+**Location of the Board:** Battery box 
+
+**List of I/O and Connections:**
+    
+    * Leaderboard Input
+        * Input from BPS Leader Board, tells the Minion board when to gather measurments form the batteries. 
+    * LeaderBoard Output 
+        * Outputs to the leaderboard, transmits voltage and temperature readings from the battery modules.
```

Comment:
Specifically, this is an Isolated SPI bus. Also, the input or output connectors can connect to the leader board or to other minion boards

Reply:
I changed it to show that they are isolated SPI bus and that they can also connect to other minion boards